azure-devops/azext_devops/dev/pipelines/pipeline.py

```diff
@@ -96,8 +101,31 @@ def pipeline_show(id=None, name=None, open=False, organization=None, project=Non
     return build_definition
 
 
+def set_param_variable(variables, as_dict=False):
+    param_variables = None
+    if variables is not None and variables:
+        param_variables = {}
+        for variable in variables:
+            separator_pos = variable.find('=')
+            if separator_pos >= 0:
+                if as_dict:
+                    param_variables[variable[:separator_pos]] = {"value": variable[separator_pos + 1:]}
+                else:
+                    param_variables[variable[:separator_pos]] = variable[separator_pos + 1:]
+            else:
+                raise ValueError('The --variables argument should consist of space separated "name=value" pairs.')
+    return param_variables
+
+
+def convert_param(parameters):
```

Comment:
as per def both variable and param are type dict in the class definition, then why this special handling for param ?

https://github.com/microsoft/azure-devops-python-api/blob/451cade4c475482792cbe9e522c1fee32393139e/azure-devops/azure/devops/v6_0/pipelines/models.py#L364-L367